application/app.py

```diff
@@ -437,18 +449,29 @@ def homography_display_results(self):
         # TODO: Nicer formatting for computed goodness images
         for i in range(self.worldPts.shape[0]):
             # world image
-            cv2.circle(worldImg, tuple(np.int32(np.round(self.worldPts[i] / self.unitPixRatio))), 2, cvBlue)
-            cv2.circle(worldImg, tuple(np.int32(np.round(projectedVideoPts[i] / self.unitPixRatio))), 2, cvRed)
-            cv2.putText(worldImg, str(i+1), tuple(np.int32(np.round(self.worldPts[i]/self.unitPixRatio)) + 5), cv2.FONT_HERSHEY_PLAIN, 2., cvBlue, 2)
+            worldDraw = ImageDraw.Draw(worldImg)
+            r = 4
+            x, y = tuple(np.int32(np.round(self.worldPts[i] / self.unitPixRatio)))
+            worldDraw.ellipse((x-r, y-r, x+r, y+r), outline=cvBlue)
+            x, y = tuple(np.int32(np.round(projectedVideoPts[i] / self.unitPixRatio)))
+            worldDraw.ellipse((x-r, y-r, x+r, y+r), outline=cvRed)
+            x, y = tuple(np.int32(np.round(self.worldPts[i]/self.unitPixRatio)) + 5)
+            worldDraw.text((x, y), str(i+1), fill=cvBlue)
+
             # video image
-            cv2.circle(videoImg, tuple(np.int32(np.round(self.videoPts[i]))), 2, cvBlue)
-            cv2.circle(videoImg, tuple(np.int32(np.round(projectedWorldPts[i]))), 2, cvRed)
-            cv2.putText(videoImg, str(i+1), tuple(np.int32(np.round(self.videoPts[i]) + 5)), cv2.FONT_HERSHEY_PLAIN, 2., cvBlue, 2)
+            videoDraw = ImageDraw.Draw(videoImg)
+            x, y = tuple(np.int32(np.round(self.videoPts[i])))
+            videoDraw.ellipse((x-r, y-r, x+r, y+r), outline=cvBlue)
+            x, y = tuple(np.int32(np.round(projectedWorldPts[i])))
+            videoDraw.ellipse((x-r, y-r, x+r, y+r), outline=cvRed)
+            x, y = tuple(np.int32(np.round(self.videoPts[i]) + 5))
+            videoDraw.text((x, y), str(i+1), fill=cvBlue)
```

Comment:
I think differentiating the text color would be useful. White text with a black stroke is visible on anything.